src/Crypto/Lol/Applications/SymmSHE.hs

```diff
@@ -221,7 +221,7 @@ modSwitchPT ct = let CT MSD k l c = toMSD ct in
 ---------- Key switching ----------
 
 type LWECtx t m' z zq =
-  (ToInteger z, Reduce z zq, Ring zq, Fact m', CElt t z, CElt t zq)
+  (ToInteger z, Reduce z zq, Ring zq, Fact m', CElt t z, CElt t zq, Random zq)
```

Comment:
I got the comments via emails.

I think the easiest solution in these cases is for you to pull in my branch
and make the edits yourself (otherwise we could go back and forth forever).
This might be an argument for me using your repo instead of mine, but if
you give me your public key, you should be able to commit directly to my
repo when you clone my branch.

On Sat, Oct 31, 2015 at 4:38 PM, cpeikert <notifications@github.com> wrote:

> In src/Crypto/Lol/Applications/SymmSHE.hs
> <https://github.com/cpeikert/Lol/pull/3#discussion_r43575955>:
>
> > @@ -221,7 +221,7 @@ modSwitchPT ct = let CT MSD k l c = toMSD ct in
> >  ---------- Key switching ----------
> >
> >  type LWECtx t m' z zq =
> > -  (ToInteger z, Reduce z zq, Ring zq, Fact m', CElt t z, CElt t zq)
> > +  (ToInteger z, Reduce z zq, Ring zq, Fact m', CElt t z, CElt t zq, Random zq)
>
> Ditto.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cpeikert/Lol/pull/3/files#r43575955>.
>